eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -209,6 +210,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             );
             $this->searchIndexer->createSearchIndex($output, (int) $iterationCount, !$commit);
         } else {
+            $io = new SymfonyStyle($input, $output);
+            $io->warning(<<<EOT
+You might encounter possible memory leaks while running the command under following circumstances:
+- enabled xdebug extension,
+- dev environment set,
+- inadequate memory_limit for big databases (recommended value: -1 that stands for non-limited memory usage),
+- too high --iteration-count value (default: 50),
+- specifying too few child processes for parallel batch operations (default: 'auto').
```

Comment:
I'm gonna disagree about language & optics here. This suggests that our software is faulty, which is not true.
How about:
> For optimal performance, before running this command, make sure that:
> - xdebug extension is disabled,
> - you're running it in "prod" environment,
> - memory limit for big databases is set to "-1" or adequately high value,
> - --iteration count is not too high (default: 50)
> - number of processes for parallel batch operations is not too low (default: 'auto' is a good choice)

Reply:
That works with me. POV ping @DominikaK. 😉

Reply:
Yes, we need to re-phrase it. @alongosz got a point.

Reply:
Agree with @alongosz, I'd suggest just a few more improvements:

>For optimal performance, before running this command, make sure that:

>- the xdebug extension is disabled,
>- you're running the command in "prod" environment,
>- memory limit for big databases is set to "-1" or an adequately high value,
>- --iteration count is low enough (default: 50)
>- number of processes for parallel batch operations is high enough (default: 'auto' is a good choice)